src/frontends/paddle/src/op/group_norm.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+#include "openvino/frontend/paddle/visibility.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+
+Output<ov::Node> reshape_channel_shaped_node_to_nchw(const Output<ov::Node>& node,
+                                                     const Output<ov::Node>& expected_rank) {
+    const auto one_const = default_opset::Constant::create(element::i64, Shape{1}, {1});
+    const auto two_const = default_opset::Constant::create(element::i64, Shape{1}, {2});
+    const auto tail_shape_rank = std::make_shared<default_opset::Subtract>(expected_rank, two_const);
+    const auto tail_shape = std::make_shared<default_opset::Broadcast>(one_const, tail_shape_rank);
+    const auto C_dim = std::make_shared<default_opset::ShapeOf>(node);
+    const auto new_shape = std::make_shared<default_opset::Concat>(OutputVector{one_const, C_dim, tail_shape}, 0);
+    return std::make_shared<default_opset::Reshape>(node, new_shape, false);
+}
+
+NamedOutputs group_norm(const NodeContext& node) {
+    auto data = node.get_input("X");
+    size_t num_groups = static_cast<size_t>(node.get_attribute<int32_t>("groups"));
+    auto epsilon = node.get_attribute<float>("epsilon", 1e-5);
+    auto data_layout = node.get_attribute<std::string>("data_layout", "NCHW");
+
+    const auto& pshape = data.get_partial_shape();
+    PADDLE_OP_CHECK(node, pshape.rank().is_static());
+    size_t rank_size = pshape.rank().get_length();
+    PADDLE_OP_CHECK(node, rank_size >= 2, "2-D and above tensors supported only");
+
+    if (data_layout == "NHWC") {
```

Comment:
What if don't we have defined layout in the model?

Reply:
In line 29, the default value of `data_layout` is "NCHW", which is same to Paddle doc description.

Reply:
@ilyachur Some Paddle operations have this data_layout attribute, while some are not. For NHWC layout, it is okay to use this layout conversion for the op mapper, I think. We could apply a transformation from model perspective to eliminate the conversions between operations later.